jobs/resource_job.go

```diff
@@ -350,7 +364,8 @@ type JobRun struct {
 	Trigger     string   `json:"trigger,omitempty"`
 	RuntType    string   `json:"run_type,omitempty"`
 
-	OverridingParameters RunParameters `json:"overriding_parameters,omitempty"`
+	OverridingParameters RunParameters  `json:"overriding_parameters,omitempty"`
+	JobParameters        []JobParameter `json:"job_parameters,omitempty"`
```

Comment:
FTR, we defined `JobParameters`, seeing that `OverridingParameters` was also defined but see no usage anywhere in the provider. There is no Terraform data source to retrieve job runs and the only place where runs are triggered/polled is when the job resource controls the job's state through `always_running` or `control_run_state`.

Reply:
IIUC this keeps this struct in sync with OpenAPI.